app/views/mediatorDetails.html

```diff
@@ -37,6 +37,17 @@
               <td>{{mediatorDetails.version}}</td>
             </tr>
             <tr>
+              <td><strong>Last Heartbeat: </strong></td>
+              <td title="{{mediatorDetails.lastHeartbeatDisplay}}">
+                <span ng-if="mediatorDetails.lastHeartbeatStatus === 'never'" class="label label-default">Never</span>
+                <span ng-if="mediatorDetails.lastHeartbeatStatus !== 'never'" class="label label-{{mediatorDetails.lastHeartbeatStatus}}" title="{{mediatorDetails.lastHeartbeatDisplayExplain}}">{{mediatorDetails.lastHeartbeatDisplay}}</span>
```

Comment:
I think instead of using `title=` we should use the tooltips that we have been using in the transaction log. Just to be more consistent. See https://github.com/jembi/openhim-console/blob/master/app/views/transactions.html#L29